CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# Version: 0.44.0
+
+* [#1816](https://github.com/apollographql/apollo-ios/pull/1816): fix(deps): update dependency gatsby-theme-apollo-docs to v4.7.9
+* [#1790](https://github.com/apollographql/apollo-ios/pull/1790): Document improvement for README about replace .xcworkspace to .xcodeproj
+* [#1817](https://github.com/apollographql/apollo-ios/pull/1817): Foundational Work For Codegen Rewrite
+* [#1766](https://github.com/apollographql/apollo-ios/pull/1766): Add a "Who is Apollo" section to the main README
+* [#1767](https://github.com/apollographql/apollo-ios/pull/1767): chore: fixed a typo
+* [#1769](https://github.com/apollographql/apollo-ios/pull/1769): Fix swift warning when using cocoapods
+* [#1778](https://github.com/apollographql/apollo-ios/pull/1778): Bump lodash from 4.17.19 to 4.17.21 in /SimpleUploadServer
+* [#1775](https://github.com/apollographql/apollo-ios/pull/1775): chore(deps): update dependency xmlhttprequest-ssl to 1.6.2 [security]
```

Comment:
i wonder if there's a way to filter out commits from dependabot....

Reply:
Yeah, there is! That's why I'm testing this right now lol

Reply:
I don't know how to deal with the fact that the bot hasn't signed the CLA though...